### PR TITLE
Project list on web site

### PR DESCRIPTION
**Pull request by cornelius**

### CODE REVIEW

File: data/projects.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+title: Projects
+introduction: |
+  These are the open source projects which are officially hosted by the
```
Comment (SergeCroise) [inline data/projects.yml:3]:
```suggestion
  These are the open-source projects which are officially hosted by the
```
see open-source as an adjective according to merriam-webster:
https://www.merriam-webster.com/dictionary/open-source

Comment (cornelius) [reply data/projects.yml:3]: Our current practice is to write it without the hyphen, especially when it's about terms which acts more as names such as "open source project".

File: data/projects.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  have a look at the questionnaire for new projects.
+
+listclass: style1 medium less-bottom-margin less-top-margin
+
+projects:
+  - publiccode: https://raw.githubusercontent.com/OpenRailAssociation/landscape/main/data/openrail/osrd.yml
+  - publiccode: https://github.com/OpenRailAssociation/landscape/raw/main/data/openrail/rcm-oss.yml
+  - publiccode: https://github.com/OpenRailAssociation/landscape/raw/main/data/openrail/dac-migration-dss.yml
+  - publiccode: https://github.com/OpenRailAssociation/landscape/raw/main/data/openrail/nge.yml
+  - publiccode: https://github.com/OpenRailAssociation/landscape/raw/main/data/openrail/liblrs.yml
```
Comment (mxmehl) [inline data/projects.yml:15]: Use final links directly, spares the build to follow a 302 redirect:

```suggestion
  - publiccode: https://raw.githubusercontent.com/OpenRailAssociation/landscape/main/data/openrail/osrd.yml
  - publiccode: https://raw.githubusercontent.com/OpenRailAssociation/landscape/main/data/openrail/rcm-oss.yml
  - publiccode: https://raw.githubusercontent.com/OpenRailAssociation/landscape/main/data/openrail/dac-migration-dss.yml
  - publiccode: https://raw.githubusercontent.com/OpenRailAssociation/landscape/main/data/openrail/nge.yml
  - publiccode: https://raw.githubusercontent.com/OpenRailAssociation/landscape/main/data/openrail/liblrs.yml
```

File: data/projects.yml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+title: Projects
+introduction: |
+  These are the open source projects which are officially hosted by the
+  OpenRail Association. They have been accepted through the incubation process.
+  If you would like to suggest a new project to the OpenRail Association please
+  have a look at the questionnaire for new projects.
```
Comment (mxmehl) [inline data/projects.yml:6]: I would link to the incubation process and the questionnaire if we already mention them here.

File: data/projects.yml [hunk 4]
```diff
@@ -0,0 +1,16 @@
+title: Projects
```
Comment (mxmehl) [inline data/projects.yml:1]: We could make this more explicit, also looks nicer on the website

```suggestion
title: OpenRail Projects
```

**Review by mxmehl — CHANGES_REQUESTED** [review]
Thanks for the work! I have a few suggestions.

Generally, I wonder whether the projects would better fit on a separate page. For now, it's OKish, but for a better overview and to allow for more background text and links (from the TC, mainly) a separate page seems fair.

File: data/projects.yml (hunk 2 above)
Comment (mxmehl) [reply data/projects.yml:15]: Also see #31, this should probably be integrated here

File: data/projects.yml (hunk 4 above)
Comment (cornelius) [reply data/projects.yml:1]: Ok, then we should probably also change the title of the member section from "Membership" to "OpenRail Members".

File: data/projects.yml (hunk 3 above)
Comment (cornelius) [reply data/projects.yml]: Good idea.

**Comment by cornelius** [comment]
> Generally, I wonder whether the projects would better fit on a separate page. For now, it's OKish, but for a better overview and to allow for more background text and links (from the TC, mainly) a separate page seems fair.

Agreed. I would keep it on the front page for now, but move to a separate page soon, when we are adding more projects. Same for the members.

File: data/projects.yml (hunk 4 above)
Comment (mxmehl) [reply data/projects.yml]: Good idea, will do it separately

**Pull request merged**